README.md

```diff
@@ -0,0 +1,135 @@
+# mobile-android-authentication
+
+Implementation of Authentication package
+
+## Installation
+
+To use Authentication in an Android Project:
+
+1. Add the following to the settings.gradle.kts
+
+```kotlin
+dependencyResolutionManagement {
+    ...
+    
+    repositories {
+        
+        ...
+        
+        maven("https://maven.pkg.github.com/govuk-one-login/mobile-android-authentication") {
+            if (file("${rootProject.projectDir.path}/github.properties").exists()) {
+                val propsFile = File("${rootProject.projectDir.path}/github.properties")
+                val props = Properties().also { it.load(FileInputStream(propsFile)) }
+                val ghUsername = props["ghUsername"] as String?
+                val ghToken = props["ghToken"] as String?
+
+                credentials {
+                    username = ghUsername
+                    password = ghToken
+                }
+            } else {
+                credentials {
+                    username = System.getenv("USERNAME")
+                    password = System.getenv("TOKEN")
+                }
+            }
+        }
+    }
+}
+```
+
+2. For local development, ensure you have a `github.properties` in the project's root which includes your username and an access token
+3. Add `implementation("uk.gov.android:authentication:_")` for latest version. Check packages for version information
+
+## Package description
+
+The Authentication package authenticates a users details and enables them to log into their account securely. This is done by providing them with a login session and token.
+
+The package integrates [openID](https://openid.net/developers/how-connect-works/) AppAuth and conforms to its standards, documentation can be found here [AppAuth](https://github.com/openid/AppAuth-Android)
+
+### Types
+
+#### LoginSessionConfiguration
```

Comment:
Do we want these sections to be within KDoc instead of in the README?

Reply:
Will leave in README but can add something to KDoc too, to be honest, they don't really shine much more light than just the member variables in the classes